### PR TITLE
Fix ``planemo shed_test``

### PR DESCRIPTION
**Pull request by mvdbeek**
And potentially speed up shed installs by not requesting the entire tool shed contents ...

### CODE REVIEW

File: planemo/shed/interface.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def find_repository(tsi, owner, name):
     """Find repository information for given owner and repository
     name.
     """
-    repos = tsi.repositories.get_repositories()
+    repos = tsi.repositories.get_repositories(owner=owner, name=name)
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply planemo/shed/interface.py:21]: (yep, also noticed some more wrong things in this test!)

File: tests/test_cmd_shed_test.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        """Test --galaxy_startup_timeout."""
+        with self._isolate():
+            test_command = self._test_command(
+                "--no_dependency_resolution",
+                "--owner",
+                "iuc",
+                "--name",
+                "collection_element_identifiers",
+                "--shed_target",
+                "https://toolshed.g2.bx.psu.edu/",
```
Comment (nsoranzo) [inline tests/test_cmd_shed_test.py:23]:
```suggestion
                "toolshed",
```

File: tests/test_cmd_shed_test.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        """Test --galaxy_startup_timeout."""
+        with self._isolate():
+            test_command = self._test_command(
+                "--no_dependency_resolution",
+                "--owner",
+                "iuc",
+                "--name",
+                "collection_element_identifiers",
+                "--shed_target",
+                "toolshed",
```
Comment (mvdbeek) [inline tests/test_cmd_shed_test.py:23]: hmm, that doesn't work of course, but I guess it should ideally.

Comment (mvdbeek) [reply tests/test_cmd_shed_test.py:23]: Should work with 3c71f5c32a1959de6d654000eca5154d95a7e463

**Pull request merged**